Marlin/Configuration.h

```diff
@@ -143,7 +143,7 @@
 #if ENABLED(SWITCHING_EXTRUDER)
   #define SWITCHING_EXTRUDER_SERVO_NR 0
   #define SWITCHING_EXTRUDER_SERVO_ANGLES { 0, 90 } // Angles for E0, E1[, E2, E3]
-  #if (EXTRUDERS == 4)
+  #if EXTRUDERS >= 3
     #define SWITCHING_EXTRUDER_E23_SERVO_NR 1
   #endif
```

Comment:
I think, it is  not nessesary to define `SWITCHING_EXTRUDER_E23_SERVO_NR ` if second extruder is not switching - nothing to switch.
Then checking shold be #if EXTRUDERS > 3 to cover the case of 5 extruders.